src/internal/components/cartesian-chart/left-labels.tsx

```diff
@@ -87,14 +90,25 @@ function LeftLabels({
                   className={clsx(styles.grid, styles.ticks_line)}
                   x1={-TICK_LENGTH}
                   y1={0}
-                  x2={width}
+                  x2={plotWidth}
                   y2={0}
                   aria-hidden="true"
                 />
               )}
-              <text className={styles.ticks__text} x={-(TICK_LENGTH + TICK_MARGIN)} y={0}>
-                {lines.join(' ')}
-              </text>
+
+              {lines.map((line, lineIndex) => {
+                const lineTextProps = {
+                  x: -(TICK_LENGTH + TICK_MARGIN),
+                  y: lineIndex * TICK_LINE_HEIGHT,
```

Comment:
This will render every line below each other starting from the original first line, but shouldn't it be better to center them vertically? I also did this [here](https://github.com/cloudscape-design/components/pull/1691/files#diff-cac1f0bfda43aa85aac8be6bb1fffe700f614c938b433eff37ae7c6ab83a0aa9R100) and I centered them by adding `- (lines.length - 1) * TICK_LINE_HEIGHT * 0.5`.

```
y={lineIndex * TICK_LINE_HEIGHT - (lines.length - 1) * TICK_LINE_HEIGHT * 0.5}
```
or it could be simplified a bit
```
y={TICK_LINE_HEIGHT * (lineIndex - (lines.length - 1) * 0.5)}
```

Reply:
This requires a design decision, I like the right alignment better. We can also check how it works for bottom labels where the multi-line is supported.

Reply:
> I like the right alignment better

But I'm talking about _vertically_ centering, so items can still be anchored to the right. [Here](https://d21d5uik3ws71m.cloudfront.net/components/5b3f734fa9edd6ba4cdc3e4108999a1e8a6d038d/index.html#/light/bar-chart/left-label-wrap) is a deployed example of how that looks like.

Reply:
Oh, I din't realise that. That's great 👍  Feel free to push a commit to the PR.

Reply:
Done :)